web/app/mu-plugins/filters.php

```diff
@@ -19,3 +19,13 @@
 	$config_contents = str_replace( 'define(', 'Config::define(', $config_contents );
 	return $config_contents;
 } );
+
+/**
+ * Update the wp-config filename to use config/application.php.
+ *
+ * @return string
+ */
+add_filter( 'pantheon.multisite.config_filename', function ( $config_filename ) {
+    $config_filename = 'config/application.php';
+    return $config_filename;
```

Comment:
```suggestion
    return 'config/application.php';
```
unnecessary assignment. optional.

Reply:
I was actually thinking about that just now, too